library/chips/src/com/android/ex/chips/RecipientEditTextView.java

```diff
@@ -349,6 +356,11 @@ public void onSelectionChanged(int start, int end) {
     }
 
     @Override
+    public boolean onPreDraw() {
+        return isCursorVisible() ? super.onPreDraw() : true;
+    }
```

Comment:
TextView#onPreDraw will scroll to the cursor even if it's not visible. Since the cursor is always at the end, this also causes the view to scroll to the bottom.